pyfar/plot/line.py

```diff
@@ -47,7 +47,10 @@ def time(signal, dB=False, log_prefix=20, log_reference=1, unit="s",
         or creates a new figure if none exists.
     style : str
         ``light`` or ``dark`` to use the pyfar plot styles or a plot style from
-        ``matplotlib.style.available``. The default is ``light``.
+        ``matplotlib.style.available``. Pass a dictonary to set specific plot
+        parameters, for example ``style = {'axes.facecolor':'black'}``. Pass an
+        empty dictonary ``style = {}`` to use the current active plotstyle.
```

Comment:
Just a tiny correction, I think it should be
```suggestion
        empty dictonary ``style = {}`` to use the currently active plotstyle.
```
sorry for not noting this earlier